ci-operator/jobs/openshift/machine-api-provider-openstack/openshift-machine-api-provider-openstack-master-presubmits.yaml

```diff
@@ -74,7 +74,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
we probably want to leave it running by default, so that we know when it's fixed. If we make it only available on demand, we're also very likely to forget to restore the job once it's fixed. We want it to run by default in MAPO since it has the potential to break the NFV job.

Reply:
ok